tests/OperationStore.spec.ts

```diff
@@ -64,21 +71,69 @@ async function constructAnchoredUpdateOperation (
     }]
   };
 
-  const updateOperationBuffer = await OperationGenerator.generateUpdateOperation(updatePayload, '#key1', privateKey);
+  const updateOperationBuffer = await OperationGenerator.generateUpdateOperationBuffer(updatePayload, '#key1', privateKey);
   return constructAnchoredOperation(updateOperationBuffer, transactionNumber, transactionTime, operationIndex);
 }
 
+const databaseName = 'sidetree';
```

Comment:
Consider expose these constants in `MongoDbOperationStore` and use them here.

Reply:
Made these public readonly static members of MongoDbOperationStore.